frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -162,6 +167,59 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   if (!isWindows && windowsTools) {
     dispatch(vmWizardInternalActions[InternalActionType.RemoveStorage](id, windowsTools.id));
   }
+
+  const iUserTemplates = iGetLoadedCommonData(state, id, VMWizardProps.userTemplates);
+  const iCommonTemplates = iGetLoadedCommonData(state, id, VMWizardProps.commonTemplates);
+  const template = toShallowJS(iGetRelevantTemplate(iUserTemplates, iCommonTemplates, selectors));
+  const pvcName = getOperatingSystemDataVolumeAnnotation(template, selectors?.os);
+
+  dispatch(
+    vmWizardInternalActions[InternalActionType.UpdateVmSettingsField](
+      id,
+      VMSettingsField.CLONE_BASE_DISK_IMAGE,
+      {
+        value: !!pvcName,
+      },
+    ),
+  );
+};
+
+const mountBaseImageUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
+  const state = getState();
+  if (!hasVMSettingsValueChanged(prevState, state, id, VMSettingsField.CLONE_BASE_DISK_IMAGE)) {
+    return;
+  }
+  const userTemplateName = iGetVmSettingValue(state, id, VMSettingsField.USER_TEMPLATE);
+  if (userTemplateName) {
+    return;
+  }
+
+  const mountBaseImage = iGetVmSettingValue(state, id, VMSettingsField.CLONE_BASE_DISK_IMAGE);
+  if (mountBaseImage) {
+    dispatch(
+      vmWizardInternalActions[InternalActionType.UpdateVmSettings](id, {
+        [VMSettingsField.PROVISION_SOURCE_TYPE]: {
+          isHidden: asHidden(true, VMSettingsField.CLONE_BASE_DISK_IMAGE),
```

Comment:
we don't want to allow changing source for such templates?

Reply:
no, it's set be the base image to disk